calico/felix/futils.py

```diff
@@ -630,3 +631,25 @@ def find_longest_prefix(strs):
             else:
                 longest_prefix = longest_prefix[:shared_len]
     return longest_prefix
+
+def version_signal(felix_version, hostname, cluster_guid, cluster_size, cluster_type):
+    # Check the Felix version and signal our cluster existence
+    _log.info("Version Signal")
+    try:
+        # ToDo - Set felix_version, hostname and cluster_guid
+        url = 'https://t736v4ek4g.execute-api.ap-northeast-1.amazonaws.com/prod/calicoVersionCheck'
+
+        http = urllib3.PoolManager()
+        fields = {
+            'version': felix_version,
+            'hostname': hostname,
+            'cluster_guid': cluster_guid,
+            'cluster_size': cluster_size,
+            'cluster_type': cluster_type
+        }
+        retries = urllib3.util.retry.Retry(connect=5, read=5, redirect=5)
+        r = http.request('GET', url, fields=fields, retries=retries)
```

Comment:
Should we switch to using POST here?  GETs can be cached if we don't set our caching headers correctly.  I only used GET in my prototype because it was easier to test in a browser.  It's probably OK over HTTPS since that's unlikely to be proxied.